tpm/structures.go

```diff
@@ -43,6 +43,11 @@ type pcrSelection struct {
 	Mask pcrMask
 }
 
+type nvAttribute struct {
```

Comment:
Although these structures are not exported and don't require comments from Go style, we have been trying to keep them commented so that developers understand the details. Please add similar comments to this and other library-private structures.